portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,6 +32,10 @@
   // Constants for certain areas of the DOM.
   private static final String COMMENT_INPUT = "comment-input";
   private static final String BOTTOM_OF_PAGE = "/index.html#connect-container";
+  
+  // Constants for Entity instances.
+  private static final String MESSAGE_ENTITY = "Message";
+  private static final String COMMENT_PROPERTY = "comment";
 
   private ArrayList<String> messages;
```

Comment:
This can be a List

Reply:
Understood! I ended up deleting it because I realized the ArrayList is unused but I'll make sure to do that in future pull requests. Thank you!